src/nodes/typeGuards/single.ts

```diff
@@ -1,122 +1,245 @@
 import * as ts from "typescript";
 
+export type AnyKeyword = ts.KeywordToken<ts.SyntaxKind.AnyKeyword>;
+export type BigIntKeyword = ts.KeywordToken<ts.SyntaxKind.BigIntKeyword>;
+export type BooleanKeyword = ts.KeywordToken<ts.SyntaxKind.BooleanKeyword>;
+export type FalseKeyword = ts.KeywordToken<ts.SyntaxKind.FalseKeyword>;
+export type ImportKeyword = ts.KeywordToken<ts.SyntaxKind.ImportKeyword>;
+export type NeverKeyword = ts.KeywordToken<ts.SyntaxKind.NeverKeyword>;
+export type NullKeyword = ts.KeywordToken<ts.SyntaxKind.NullKeyword>;
+export type NumberKeyword = ts.KeywordToken<ts.SyntaxKind.NumberKeyword>;
+export type ObjectKeyword = ts.KeywordToken<ts.SyntaxKind.ObjectKeyword>;
+export type StringKeyword = ts.KeywordToken<ts.SyntaxKind.StringKeyword>;
+export type SuperKeyword = ts.KeywordToken<ts.SyntaxKind.SuperKeyword>;
+export type SymbolKeyword = ts.KeywordToken<ts.SyntaxKind.SymbolKeyword>;
+export type ThisKeyword = ts.KeywordToken<ts.SyntaxKind.ThisKeyword>;
+export type TrueKeyword = ts.KeywordToken<ts.SyntaxKind.TrueKeyword>;
+export type UndefinedKeyword = ts.KeywordToken<ts.SyntaxKind.UndefinedKeyword>;
+export type UnknownKeyword = ts.KeywordToken<ts.SyntaxKind.UnknownKeyword>;
+export type VoidKeyword = ts.KeywordToken<ts.SyntaxKind.VoidKeyword>;
```

Comment:
As TypeScript already defines quite a few types for "Node Keywords", this PR simply adds the missing ones and corresponding type guards. I added these mostly just for completeness, but there are a few use cases where you might need to know if a node represents a keyword.

For example, in #50 a lot of these functions are needed (well not *needed* needed as we could always just manually compare `node.kind` with `ts.SyntaxKind.XKeyword`). But if users want to define similar functions, having these available to them will make things easier.

Reply:
Note: In my option, these types would be better named `XKeywordNode` rather than just `XKeyword`, but that would go against TypeScript's current naming for such types. Maybe it's worth opening an issue for this on TypeScript to rename such types?

Reply:
> adds the missing ones and corresponding type guards

Makes sense! I like it 😄 thanks for the explanation!

> worth opening an issue for this on TypeScript

Yeah agreed. All you! 😁